src/pages/explorePage/explorePage.jsx

```diff
@@ -3,8 +3,11 @@ import './explorePage.css';
 import ContentFilter from '@/components/contentFilter/contentFilter';
 import AllNews from '@/components/allNews/allNews';
 import SearchPanel from '@/components/searchPanel/searchPanel';
-
```

Comment:
Would you be able to make corrections in files unrelated to the current task in a separate PR? It becomes too hard to review. It is like a porridge.